practice.js

```diff
@@ -1,14 +1,26 @@
+//Importing all the common nodes needed throughout the code
 let input = document.getElementById("input");
 let div = document.getElementById("result");
+let add = document.getElementById("add");
```

Comment:
Give proper name to variables. add ==> addButton or addBtn

Reply:
Changed variable names according as suggested.